frontend/src/app/hooks/useStateModal.tsx

```diff
@@ -62,6 +62,8 @@ function useStateModal({ initState }: { initState?: any }) {
             Object.create(null),
             ...spreadParmas,
           );
+          // @TM 清空
```

Comment:
That's great, the modal should be recreated after close, but...so, it is better to clear the ref state, 👍🏿 .
Additional, could you remove the `todo` comment on this? the fix is ok for me.

Reply:
> That's great, the modal should be recreated after close, but...so, it is better to clear the ref state, 👍🏿 . Additional, could you remove the `todo` comment on this? the fix is ok for me.

ok